content/en/blog/_posts/2025-xx-xx-ippr-goes-ga.md

```diff
@@ -0,0 +1,125 @@
+and `limits` in the Pod's specification by utilizing the new `resize` subresource.
+
+## How can I start using In-Place Pod Resize?
+
+Detailed usage instructions and examples are provided in the official documentation:
+[Resize CPU and Memory Resources assigned to Containers](/docs/tasks/configure-pod-container/resize-container-resources/).
+
+## How does this help me?
+
+In-place Pod Resize is a foundational building block that unlocks seamless, vertical autoscaling and
+improvements to workload efficiency.
+
+* **Resources adjusted without disruption** Workloads sensitive to latency or restarts can have their resources 
+modified in-place without downtime or loss of state.
+* **More powerful autoscaling** Autoscalers are now empowered to adjust resources and with less
+impact. For example, Vertical Pod Autoscaler (VPA)'s `InPlaceOrRecreate` update mode, which leverages this
+feature, has graduated to beta. This allows resources to be adjusted automatically and seamlessly based on
+usage with minimal disruption.
+  * See [AEP-4016](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/enhancements/4016-in-place-updates-support) for more details.
+* **Address transient resource needs** Workloads that temporarily need more resources can be adjusted quickly. This enables features like the CPU Startup Boost ([AEP-7862](https://github.com/kubernetes/autoscaler/pull/7863)) where applications can request more CPU during startup and then automatically scale back down.
+
+Here are a few examples of some use cases:
+
+* A game server that needs to adjust its size with shifting player count.
+* A pre-warmed worker that can be shrunk while unused but inflated with the first request.
+* Dynamically scale with load for efficient bin-packing.
+* Increased resources for JIT compilation on startup.
+
+## Changes between Beta (1.33) and Stable (1.35)
+
+Since the initial beta in v1.33, development effort has primarily been around stabilizing the feature and
+improving its usability based on community feedback. Here are the primary changes for the stable release:
+
+### User-facing changes
+
+* **Memory limit decrease** Decreasing memory limits was previously prohibited. This restriction has been
+lifted, and memory limit decreases are now permitted. The Kubelet attempts to prevent OOM-kills by allowing the
+resize only if the current memory usage is below the new desired limit. However, this check is best-effort and
+not guaranteed.
+* **Prioritized resizes** If a node doesn't have enough room to accept all resize requests, _Deferred_ resizes 
+are reattempted based on the following priority:
+  * PriorityClass
+  * QoS class
+  * Duration _Deferred_, with older requests prioritized first.
+* **Pod Level Resources (Alpha)** Support for in-place Pod Resize with Pod Level Resources has been introduced 
+behind its own feature gate, which is alpha in v1.35.
+
+### Observability
```

Comment:
nit: remove this heading, and just include the observability changes in the above list (they're user facing enough).

Reply:
done!